ALCameraViewController/ViewController/ALCameraViewController.swift

```diff
@@ -91,15 +132,169 @@ public class ALCameraViewController: UIViewController {
     public override func preferredStatusBarUpdateAnimation() -> UIStatusBarAnimation {
         return UIStatusBarAnimation.Slide
     }
+  
+
+    public override func loadView() {
+        super.loadView()
+        self.view.backgroundColor = UIColor.blackColor()
+        self.view.addSubview(cameraView)
+        self.view.addSubview(cameraOverlay)
+        self.view.addSubview(cameraButton)
+        self.view.addSubview(libraryButton)
+        self.view.addSubview(closeButton)
+        self.view.addSubview(swapButton)
+        self.view.addSubview(flashButton)
+        self.view.setNeedsUpdateConstraints()
+    }
```

Comment:
Nice n sleek, you could even trim it down to 
`[viewOne, viewTwo].forEach { self.view.addSubview($0) }`

But thats just me being pedantic ;)